gnucash/gnome/gnc-plugin-page-invoice.c

```diff
@@ -350,6 +322,8 @@ gnc_plugin_page_invoice_new (InvoiceWindow *iw)
     return plugin_page;
 }
 
+G_DEFINE_TYPE_WITH_PRIVATE(GncPluginPageInvoice, gnc_plugin_page_invoice, GNC_TYPE_PLUGIN_PAGE_INVOICE)
+
```

Comment:
```G_DEFINE_TYPE_WITH_PRIVATE(GncPluginPageInvoice, gnc_plugin_page_invoice, GNC_TYPE_PLUGIN_PAGE_INVOICE)```
There's the problem: You're declaring the class as its own parent. The line should be ```G_DEFINE_TYPE_WITH_PRIVATE(GncPluginPageInvoice, gnc_plugin_page_invoice, GNC_TYPE_PLUGIN_PAGE)```

Reply:
Yes, I see. You are right. I thought that was the class itself. Need to look through the other changes about that as well. 

https://developer.gnome.org/gobject/stable/gobject-Type-Information.html#G-DEFINE-TYPE-WITH-PRIVATE:CAPS

the last parameter is T_P | The GType of the parent type

Reply:
works ! compiles and loads.

Reply:
also did the trick fpr qif import